Doctrine/Phpcr/AutoRouteListener.php

```diff
@@ -73,6 +75,14 @@ public function onFlush(ManagerEventArgs $args)
                     $dm->persist($autoRoute);
                     $uow->computeChangeSets();
                 }
+
+                // reset locale to the original locale
+                if (null !== $locale) {
+                    $meta = $dm->getMetadataFactory()->getMetadataFor(get_class($document));
+                    $dm->findTranslation($meta->getName(), $meta->getIdentifierValue($document), $locale);
```

Comment:
as with the method to determine the locale, we where pondering to expose
it in the document manager as well, to make it more "official"